### ISSUE TITLE
Any way to skip host key verification (BadHostKeyException)?

### ISSUE THREAD

**Issue by brabiega** [paramiko/paramiko#1045]
Hello guys,

Is there any way to skip host key verification?
Let's say I'm 100% aware what I'm doing and I want to connect to host which changed it's key fingerprint.

Currently ```connect``` function will raise ```BadHostKeyException```
http://docs.paramiko.org/en/2.2/api/ssh_exception.html#paramiko.ssh_exception.BadHostKeyException

I don't see any way not special policy to skip such check. Any tips?

**Comment by ploxiln** [comment]
If you're using SSHClient `connect()`, then don't call `load_host_keys()`, and use something like the WarningPolicy, see http://docs.paramiko.org/en/2.2/api/client.html#paramiko.client.SSHClient.set_missing_host_key_policy

**Comment by bitprophet** [comment]
Yup, this is a policy-setting thing, same as in regular OpenSSH (where you would set a config option).

**Comment by brabiega** [comment]
Thanks. Seems good :+1:
